lib/api/controllers/server.js

```diff
@@ -122,41 +122,48 @@ class ServerController extends NativeController {
         err => errorsManager.get('core', 'fatal', 'service_unavailable', err),
       result = { status: 'green', services: {} };
 
-    try {
-      await this.kuzzle.cacheEngine.internal.info();
-      result.services.internalCache = 'green';
+    let services;
+    if (request.input.args && request.input.args.services) {
```

Comment:
`request.input.args` will always be present, and you should check the type of `services` instead to avoid an error if somebody send an array for example